src/test/java/com/nimbusds/jose/crypto/impl/ConcatKDFTest.java

```diff
@@ -65,7 +66,42 @@ public void testComposeOtherInfo() {
 		
 		assertTrue(Arrays.equals(expected, otherInfo));
 	}
-	
+
+	public void testComposeOtherInfoWithTag() {
```

Comment:
Should we add other tests using examples from the Spec 9https://datatracker.ietf.org/doc/html/draft-madden-jose-ecdh-1pu-04#appendix-B)? For example, to check all generated values in examples there (Ze, Zs, Z, derived key etc.)?

Reply:
Definitively, since we have test-vectors, we should be using them.